DevOps/Contribute/Readme.md

```diff
@@ -26,3 +26,4 @@
 - [rclone](https://github.com/rclone)
 - [Stemformatics](https://github.com/wellslab/s4m-api)
 - [Wireshark](https://gitlab.com/wireshark/wireshark)
+- [Kubescape](https://github.com/kubescape/kubescape)
```

Comment:
Please add name according to the alphabetical order

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
Name added according to the alphabetical order.
Please review.